Sources/String+Sweetness.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public extension String {
+    public var isValidEmail: Bool {
+        let emailRegEx = ".*@.*"
```

Comment:
Backend should accept any valid email. Only regex that matches all valid emails that I know of is the top one. If you've come across one that implements the whole email RFC, please link. 😄